### PR TITLE
CB-9903 added 5.4.0 tools release blog

### PR DESCRIPTION
**Pull request by stevengill**

### CODE REVIEW

File: www/_posts/2015-11-04-tools-release.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  * If you have `plugman` installed:
+
+        npm install -g plugman@latest
+
+<!--more-->
+# Changes include:
+
+## Platform updates
+When adding these platforms to your project, the following versions are now used by default.
+These platform versions were released recently, and the tools' defaults were updated:
```
Comment (sgrebnov) [inline www/_posts/2015-11-04-tools-release.md]: I don't think pinned iOS version has been changed since last release

**Comment by sgrebnov** [comment]
lgtm

File: www/_posts/2015-11-04-tools-release.md (hunk 1 above)
Comment (stevengill) [inline www/_posts/2015-11-04-tools-release.md]: You are right of course. I just wanted to highlight that 3.9.2 ios is out. 

Tools defaults weren't updated but platform versions were released. No big deal here.

Comment (sgrebnov) [inline www/_posts/2015-11-04-tools-release.md]: Makes sense +1:

**Comment by stevengill** [comment]
oo good point about the cordova-app-hello-world update. I'll add it now

**Pull request closed without merging**